v19.2/learn-cockroachdb-sql.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: Learn CockroachDB SQL
-summary: Learn some of the most essential CockroachDB SQL statements.
+title: Learn CockroachDB SQL with a Running Cluster
```

Comment:
Hm, from looking at the auto-built test site, it looks like this file is an orphan from the TOC now.  I left a comment above on the TOC JSON that is basically: I don't think we should add a new "Learn CRDB SQL MovR" page, I think we should edit the existing page with the MovR updates.  Reasoning is explained in that comment.

Reply:
Done.